src/plugins/intel_cpu/tests/unit/CMakeLists.txt

```diff
@@ -14,15 +14,18 @@ endif()
 
 if(NOT (ARM OR AARCH64))
     list(APPEND EXCLUDED_SOURCE_PATHS_FOR_UNIT_TEST
-      ${CMAKE_CURRENT_SOURCE_DIR}/transformations/arm)
+      ${CMAKE_CURRENT_SOURCE_DIR}/transformations/arm
+      ${CMAKE_CURRENT_SOURCE_DIR}/snippets_transformations/arm)
 endif()
 
 if(NOT X86_64)
     list(APPEND EXCLUDED_SOURCE_PATHS_FOR_UNIT_TEST
       ${CMAKE_CURRENT_SOURCE_DIR}/jit_kernel_test.cpp
       ${CMAKE_CURRENT_SOURCE_DIR}/registers_pool.cpp
       ${CMAKE_CURRENT_SOURCE_DIR}/transformations/x64
-      ${CMAKE_CURRENT_SOURCE_DIR}/snippets_transformations
+      ${CMAKE_CURRENT_SOURCE_DIR}/snippets_transformations/x64
+      ${CMAKE_CURRENT_SOURCE_DIR}/snippets_transformations/lowered
+      ${CMAKE_CURRENT_SOURCE_DIR}/snippets_transformations/common
```

Comment:
Can we validate `common` and `lowered` tests on arm as well?

Reply:
Applied according to details of offline discussion. Now the hierarchy of the directory is as follow. Thanks Alexandra!
```
snippets_transformations --- common --- mul_add_to_fma.cpp
                          |- x64 --------- enforce_precisions.cpp
                          |                       fq_test.cpp
                          |                       snippets_mark_skipped.cpp
                          |                       lowered ----- buffer_allocation.cpp
                          | - arm -------- snippets_mark_skipped.cpp
```